WcaOnRails/app/controllers/users_controller.rb

```diff
@@ -30,9 +30,11 @@ def index
         elsif params[:sort]
           @users = @users.order(params[:sort] => params[:order])
         end
+        users_array = @users.to_a
```

Comment:
Does this mean we'll now slurp the entire users array into memory, which we had not done previously? Is that potentially wasteful if the mysql query matches 1000s of rows, but we're limiting to much less than that?